src/test/scala/centaur/CallCacheSpec.scala

```diff
@@ -26,6 +28,18 @@ class CallCacheSpec extends FlatSpec with Matchers with ParallelTestExecution {
     }
   }
 
+  "dockerHashNoLookup" should "not lookup docker tag if backend doesn't support docker" in {
+    Workflow.fromPath(DockerHashNoLookup) match {
+      case Valid(w) =>
+        if (w.backends forall CromwellBackendsCompanion.supportedBackends.contains){
+          TestFormulas.runSequentialCachingWorkflows(w, w).run.get
+        } else {
+          succeed
```

Comment:
I think you can choose the verb depending on backend. E.g. something like:

```
testHeader = if (w.backends forall CromwellBackendsCompanion.supportedBackends.contains) {
  "dockerHashNoLookup" should "not lookup docker tag if backend doesn't support docker" in _
} else {
  "dockerHashNoLookup" should "not lookup docker tag if backend doesn't support docker" ignore _
}

testHeader {
  // Test contents
}
```

NB, didn't test this. Those `_`s might not be necessary. Or might not work at all.